### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by InterAl**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
View storybook at: https://wix-wix-ui-icons-common-pr-479.surge.sh

**Comment by WixBuildServer** [comment]
View storybook at: https://wix-wix-ui-core-pr-479.surge.sh

File: packages/wix-ui-core/src/components/Slider/Slider.spec.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import {Slider} from './index';
 import * as sinon from 'sinon';
 import {mount} from 'enzyme';
 import {Simulate} from 'react-dom/test-utils';
+import {findDOMNode} from 'react-dom';
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Slider/Slider.spec.tsx]: unused var

Comment (InterAl) [reply packages/wix-ui-core/src/components/Slider/Slider.spec.tsx]: Fixed.

File: packages/wix-ui-core/src/components/Slider/Slider.spec.tsx [hunk 2]
```diff
@@ -476,6 +476,29 @@ describe('Slider', () => {
...
+  it('should blur', () => {
+    const onBlur = sinon.spy();
+
+    const wrapper = mount(<Slider onBlur={onBlur}/>);
+
+    (wrapper.instance() as any).focus();
+    (wrapper.instance() as any).blur();
+
+    expect(document.activeElement).not.toEqual(wrapper.getDOMNode());
+    sinon.assert.called(onBlur);
```
Comment (argshook) [inline packages/wix-ui-core/src/components/Slider/Slider.spec.tsx]: im not familiar with sinon asserts, what does this return? no need to expect it? if onBlur wasn't called, will this just throw?

Comment (InterAl) [reply packages/wix-ui-core/src/components/Slider/Slider.spec.tsx]: Yes, it will throw.

**Pull request merged**